user_docs/en/userGuide.t2t

```diff
@@ -3487,27 +3487,27 @@ Please see the display's documentation for descriptions of where these keys can
 | control+shift+leftArrow key | dot2+dot8+space+f1 |
 | control+leftArrow key | dot2+space |
 | shift+alt+leftArrow key | dot2+dot7+f1 |
-| alt+leftArrow key | dot2+dot7 |
+| alt+leftArrow key | dot2+dot7+space |
 | rightArrow key | dot6+space, leftSideRightArrow |
 | control+shift+rightArrow key | dot5+dot8+space+f1 |
 | control+rightArrow key | dot5+space |
 | shift+alt+rightArrow key | dot5+dot7+f1 |
-| alt+rightArrow key | dot5+dot7 |
+| alt+rightArrow key | dot5+dot7+space |
 | pageUp key | dot1+dot2+dot6+space |
 | control+pageUp key | dot1+dot2+dot6+dot8+space |
 | upArrow key | dot1+space, leftSideUpArrow |
 | control+shift+upArrow key | dot2+dot3+dot8+space+f1 |
 | control+upArrow key | dot2+dot3+space |
 | shift+alt+upArrow key | dot2+dot3+dot7+f1 |
-| alt+upArrow key | dot2+dot3+dot7 |
+| alt+upArrow key | dot2+dot3+dot7+space |
 | shift+upArrow key | leftSideScrollDown+space |
 | pageDown key | dot3+dot4+dot5+space |
 | control+pageDown key | dot3+dot4+dot5+dot8+space |
 | downArrow key | dot4+space, leftSideDownArrow |
 | control+shift+downArrow key | dot5+dot6+dot8+space+f1 |
 | control+downArrow key | dot5+dot6+space |
 | shift+alt+downArrow key | dot5+dot6+dot7+f1 |
-| alt+downArrow key | dot5+dot6+dot7 |
+| alt+downArrow key | dot5+dot6+dot7+space |

```

Comment:
Can you please wrap keys in changed lines with code formatting syntax?
Also "key" can be dropped from these

```suggestion
| ``alt+downArrow`` | ``dot5+dot6+dot7+space`` |
```